portal/templates/redesign/teach_new/onboarding_print.html

```diff
@@ -74,62 +93,64 @@ <h4 class="col-sm-offset-2">Everything you need to start coding with your class
         <img title="Warning" alt="Warning sign" src='{% static "portal/img_new/warning.png" %}'>
         <p class="small-text"><i>This is the only time you will be able to view this page. You can print PDF format reminder cards that populate these details which can
             be given to the students.</i></p>
-        <button class="button--regular button-primary button--primary--navigation">Print reminder cards</button>
-        <a href="{% url 'onboarding5' %}" class="button--regular button--secondary button--secondary--dark">Finish setup</a>
+        <form target="_blank" method='post' action="{% url 'teacher_print_reminder_cards' class.access_code %}">
+            {% csrf_token %}
+            <input type='hidden' name='data' value='{{ query_data }}'/>
+            <div class='section'>
+                <button class='button--regular button-primary button--primary--navigation'
+                        type='submit'>Print reminder cards</button>
+                <a href="{% url 'onboarding-complete' %}" class="button--regular button--secondary button--secondary--dark">Finish setup</a>
+            </div>
+        </form>
+    </div>
+</div>
+
+<div class="sub-nav sub-nav--hidden sub-nav--warning">
+    <div class="row">
+        <img src='{% static "portal/img_new/warning.png" %}'>
```

Comment:
If you're going to have an inline img, it needs alt text.  But it would be better if it weren't inline.